### ISSUE TITLE
add ad slide section in home page to show ads for the sponsors

### ISSUE THREAD

**Issue by njhpca** [njhpca/njhpca.github.io#1]
There are 5 - 10 companies. Each company may have the following information:
Name and logo, ad statement(less than 30 words), company website link, 
One of the options is to add a slide show section as the below screen.
Another option is to use 20-30% of the right section on each main page.

A sample company info
Name: Four Sisters Winery				
ad statement: A New Jersey Local Winery.	
logo: https://www.foursisterswinery.com/wp-content/uploads/2013/01/4SISTERSwithText-245x204.jpg
website: https://www.foursisterswinery.com/		

The list is not finalized yet. The first upload can be done after we have 2-3 companies available in the first week of Sep.

**Comment by njhpca** [comment]
![image](https://github.com/njhpca/njhpca.github.io/assets/118034248/b39629f6-a406-47c7-a9ac-a7311aac7b14)

**Comment by boyang16** [comment]
I added a blank bar that can change to whatever ad is needed. Is there a specific ad that needs to be on the website now?

**Comment by houyang** [comment]
yes. There are 5-10 ads from different sponsors with their logo linked to their own websites

获取 Outlook for Android<https://aka.ms/ghei36>
________________________________

**Comment by boyang16** [comment]
what is the list of sponsors you want shown on the website

**Comment by houyang** [comment]
Sponsors for Ad


1. [高登保险](https://youtu.be/65F0kikq-iY?si=vFsLI3F3_26a616q)

2. 柯奇律师事务所  
3. [Four Sisters Winery](https://www.foursisterswinery.com/)

**Comment by houyang** [comment]
It does not look like we are going to have many sponsors right now. To simplify, you can add a fixed ad section rather than slide section above the footer(Above Donation link). Maybe 2 rows X 3cols for large/medium screen, 5 rows X 1col for small screen.
